src/complete_test.cpp

```diff
@@ -3302,11 +3260,22 @@ int main(int argc, char **argv) {
   time_step_client.call(time_step_srv);
 
   // test restart_controller
+  supervisor_get_from_def_srv.request.name = "ANOTHER_ROBOT";
+  supervisor_get_from_def_client.call(supervisor_get_from_def_srv);
+  uint64_t another_robot_node = 0;
+  if (supervisor_get_from_def_srv.response.node != 0) {
+    ROS_INFO("Got from DEF node: %ld.", supervisor_get_from_def_srv.response.node);
```

Comment:
I would have said "Got node %ld from DEF." instead.

Reply:
As well as this one?

Reply:
Should be fixed in #91.